### PR TITLE
add kor README

### PR DESCRIPTION
**Pull request by nadongjun**
Add the Korean (kor) version of README.

preview: https://github.com/nadongjun/PcapPlusPlus/blob/kor/translation/README-kor.md

### CODE REVIEW

**Comment by tigercosmos** [comment]
@nadongjun  Thanks for the contribution. Since no one on the maintainer team can speak Korean, could you find a friend to help review the wording of the PR?

File: translation/README-kor.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+[![X 팔로우](https://img.shields.io/badge/follow-%40seladb-1DA1F2?logo=x&style=social)](https://x.com/intent/follow?screen_name=seladb)
+[![GitHub 리포지토리 별표](https://img.shields.io/github/stars/seladb/PcapPlusPlus?style=social)]()
+
+</div>
+
+[PcapPlusPlus](https://pcapplusplus.github.io/)는 네트워크 패킷을 캡처, 분석 및 생성하기 위한 멀티 플랫폼 C++ 라이브러리입니다. 이 라이브러리는 효율적이고 강력하며 사용이 용이하도록 설계되었습니다.
+
+PcapPlusPlus는 다양한 네트워크 프로토콜을 디코딩하고 생성할 수 있는 기능을 제공하며, [libpcap](https://www.tcpdump.org/), [WinPcap](https://www.winpcap.org/), [Npcap](https://nmap.org/npcap/), [DPDK](https://www.dpdk.org/), [eBPF AF_XDP](https://www.kernel.org/doc/html/next/networking/af_xdp.html), [PF_RING](https://www.ntop.org/products/packet-capture/pf_ring/) 등과 같은 유용한 패킷 처리 엔진을 위한 손쉬운 C++ 래퍼를 지원합니다.
+
+번역: [English](../README.md) · [正體中文](./translation/README-zh-tw.md) · 한국어
```
Comment (seladb) [inline translation/README-kor.md:20]: Can you please fix the link to the zh-tw translation? The link is broken...

**Comment by seladb** [comment]
Thank you so much @nadongjun for adding this translation! 🙏 

Do you have a Korean-speaking colleague or friend who can review this PR? Unfortunately I don't speak Korean so I can't review the translation

File: translation/README-kor.md (hunk 1 above)
Comment (tigercosmos) [reply translation/README-kor.md:20]: I guess it should be `./README-zh-tw.md)` since all translations are in the same directory.

**Comment by hyejeong0807** [comment]
@seladb @tigercosmos Hello, I’m a Korean friend of @nadongjun. The above content([README-kr.md](http://readme-kr.md/)) has been accurately translated into Korean.

**Comment by seladb** [comment]
> @seladb @tigercosmos Hello, I’m a Korean friend of @nadongjun. The above content([README-kr.md](https://github.com/nadongjun/PcapPlusPlus/blob/kor/translation/README-kor.md)) has been accurately translated into Korean.

Thank you @hyejeong0807 for reviewing! 🙏

**Comment by tigercosmos** [comment]
@hyejeong0807  thanks! @nadongjun  could you fix the above comment about the link, other than that, the PR looks good to me.

**Comment by nadongjun** [comment]
@tigercosmos  Sure, I've made the changes.

**Review by tigercosmos — APPROVED** [review]

**Comment by seladb** [comment]
@nadongjun one more thing we missed is adding a link from [README-zh-tw.md](https://github.com/nadongjun/PcapPlusPlus/blob/kor/translation/README-zh-tw.md) to the new Korean version

**Comment by nadongjun** [comment]
@seladb Sure, I've made the changes and also added the WireGuard protocol in README-zh-tw.md.

**Comment by tigercosmos** [comment]
> @seladb Sure, I've made the changes and also added the WireGuard protocol in README-zh-tw.md.

Nice catch, @nadongjun!

@seladb, I think we need to be careful when there is any update in the README file. It's easy to forget to update translations.

**Comment by seladb** [comment]
> > @seladb Sure, I've made the changes and also added the WireGuard protocol in README-zh-tw.md.
> 
> Nice catch, @nadongjun!
> 
> @seladb, I think we need to be careful when there is any update in the README file. It's easy to forget to update translations.

@tigercosmos yes you're right. I'm not sure if there's a way to automate that 🤔

**Review by seladb — APPROVED** [review]

**Comment by seladb** [comment]
Thank you so much @nadongjun for working on this, much appreciated! 🙏

**Pull request merged**